packages/unchained-client/src/evm/avalanche/parser/index.ts

```diff
@@ -0,0 +1,16 @@
+import { ASSET_REFERENCE, toAssetId } from '@shapeshiftoss/caip'
+
+import { AvalancheTx } from '../../../generated/avalanche'
+import { BaseTransactionParser, TransactionParserArgs } from '../../parser'
+
+export class TransactionParser extends BaseTransactionParser<AvalancheTx> {
+  constructor(args: TransactionParserArgs) {
+    super(args)
+
+    this.assetId = toAssetId({
+      chainId: this.chainId,
+      assetNamespace: 'slip44',
+      assetReference: ASSET_REFERENCE.AvalancheC
+    })
```

Comment:
```suggestion
    this.assetId = avalancheAssetId
```
?

Reply:
If we only care about mainnet we certainly can. There was a bit of work put into supporting testnet (ropsten) logic, but we don't really have it through the stack obv